src/core/audio/AudioFileFlac.cpp

```diff
@@ -94,7 +96,10 @@ void AudioFileFlac::writeBuffer(surroundSampleFrame const* _ab, fpp_t const fram
 		{
 			for(ch_cnt_t channel=0; channel<channels(); ++channel)
 			{
-				buf[frame*channels() + channel] = _ab[frame][channel] * master_gain;
+				// Clip the negative side to -0.999999 in order to prevent it from changing sign
+				// Upstream issue: https://github.com/erikd/libsndfile/issues/309
+				// When this commit is reverted libsndfile-1.0.29 must be made a recuirement for FLAC
+				buf[frame*channels() + channel] = qMax(-0.999999f, _ab[frame][channel] * master_gain);
```

Comment:
`std::nextafterf(-1.0f, 0.0f)` from `<cmath>` is a portable alternative of the constant.

Reply:
Cool! I see that I used nextafter() instead of nextafterf(). They seem to be the same though.

Reply:
Same-ish. Changing to nextafterf().